src/Hl7.Fhir.Specification.Tests/Validation/BasicValidationTests.cs

```diff
@@ -311,6 +311,27 @@ public void ValidatesMultiplePossibleTypeRefs()
         }
 
 
+        [Fact]
+        public void DoNotFollowRefsSuppressesWarning()
+        {
+            var validator = new Validator(new ValidationSettings { ResourceResolver = _source, ResolveExteralReferences = true });
+
+            Patient p = new Patient();
+            p.Active = true;
+            p.ManagingOrganization = new ResourceReference("http://isure.cannot.be.found.nl/fhir/Patient/1");
```

Comment:
Text error: isure must be issue. Or better: reference could not be found?

Reply:
yep, indeed.